File: shortest_fastest_foobar/foobar.js
// var user_input = prompt("Give me a number: ");

user_input = 25;



for (var num = 0; num <= user_input; num++) {
    var show = '';
    if (num % 3 === 0) show += 'Fizz';
    if (num % 5 === 0) show += 'Buzz';
    console.log(show ? show : num);

}

File: shortest_fastest_foobar/foobar.py
def main():
    x = int(input("Enter any number: "))
    if x % 3 == 0:
        if x % 5 == 0:
            print("FizzBuzz")
        else:
            print("Fizz")
    if x % 5 == 0:
        print("Buzz")
        
        
main()
